shell/platform/darwin/ios/framework/Source/FlutterObservatoryPublisher.mm

```diff
@@ -4,17 +4,39 @@
 
 #define FML_USED_ON_EMBEDDER
 
-#import <TargetConditionals.h>
+#import "FlutterObservatoryPublisher.h"
 
-// NSNetService works fine on physical devices, but doesn't expose the services to regular mDNS
-// queries on the Simulator.  We can work around this by using the lower level C API, but that's
-// only available from iOS 9.3+/macOS 10.11.4+.
-#if TARGET_IPHONE_SIMULATOR
-#include <dns_sd.h>  // nogncheck
-#include <net/if.h>  // nogncheck
-#endif               // TARGET_IPHONE_SIMULATOR
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_RELEASE || \
+    FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_DYNAMIC_RELEASE
```

Comment:
Does `FLUTTER_RUNTIME_MODE_DYNAMIC_RELEASE` still exist? Should this be `FLUTTER_RUNTIME_MODE_JIT_RELEASE`? It looks like `FLUTTER_RELEASE` might do the right thing. That is:
```
#if FLUTTER_RELEASE
...
#else
...
#endif
```

Reply:
Ahh nice one. No, I don't think dynamic is anymore.